cmd/capabilities_wrapper/capset_other.go

```diff
@@ -1,3 +1,4 @@
+//go:build !linux
 // +build !linux
```

Comment:
Lol, this should have been in my Go 1.17 PR. But my `make golint` didn't check it because I have GOOS=linux, so golint ignored that file!

Reply:
That really makes me feel much, much better. 🙂